client/src/rr/dnssec/keypair.rs

```diff
@@ -241,11 +242,12 @@ impl KeyPair {
     /// # Return
     ///
     /// the KEY record data
+    #[allow(deprecated)]
     pub fn to_sig0key(&self, algorithm: Algorithm) -> DnsSecResult<KEY> {
         self.to_public_bytes()
             .map(|bytes| {
                      KEY::new(Default::default(),
-                              Default::default(),
+                              KeyUsage::Zone,
```

Comment:
I think that makes sense to me. Perhaps an additional method would be best though:

```rust
#[deprecated] // with some interesting explanation on deprecation
fn to_sig0key_with_usage(
      &self,
      algortihm: Algorithm,
      usage: KeyUsage) -> DnsSecResult<KEY> {
   ... existing logic with usage...
}
```

```rust
fn to_sig0key(&self, algortihm: Algorithm) -> DnsSecResult<KEY> {
   to_sig0key_with_usage(... old defaults...)
}
```